src/ClientData/include/ClientData/OwnsCaptureData.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_OWNS_CAPTURE_DATA_H_
+#define CLIENT_DATA_OWNS_CAPTURE_DATA_H_
+
+#include <functional>
+#include <memory>
+#include <optional>
+
+#include "ClientData/CaptureData.h"
+#include "OrbitBase/Logging.h"
+
+namespace orbit_client_data {
+
+class OwnsCaptureData {
```

Comment:
What about `CaptureDataHolder`? I think this would fit better into our style of naming, but I also don't feel strongly about it?

Reply:
You're right. A name of an interface/class should be a noun. So at least, it should be `CaptureDataOwner`.

Reply:
... and perhaps, later it will not exactly own CaptureData.